compiler/rustc_expand/src/base.rs

```diff
@@ -1132,11 +1132,13 @@ impl<'a> ExtCtxt<'a> {
     ///
     /// Compilation will be stopped in the near future (at the end of
     /// the macro expansion phase).
+    #[track_caller]
```

Comment:
should we just go thru and add it to like all of them?

`create_err`, `emit_err`?

Reply:
`span_bug` and `bug` don't have `#[track_caller]`, so this does not matter.

For `struct_span_err`, `create_err` and `emit_err` I've added `#[track_caller]`, good call, thanks!